platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultIgnoredConfigurationInputs.kt

```diff
@@ -20,28 +20,29 @@ import org.gradle.internal.cc.impl.initialization.ConfigurationCacheStartParamet
 import org.gradle.internal.nativeintegration.filesystem.FileSystem
 import org.gradle.internal.service.scopes.Scope.BuildTree
 import org.gradle.internal.service.scopes.ServiceScope
-import org.gradle.util.internal.GFileUtils
 import java.io.File
 import java.util.EnumSet
 import javax.inject.Inject
 
 
 @ServiceScope(BuildTree::class)
+internal
 class DefaultIgnoredConfigurationInputs(
     ignoredPathsString: String?,
     isCaseSensitive: Boolean,
-    private val rootDirectory: File
+    private val relativePaths: ConfigurationCacheRelativePaths,
```

Comment:
💭 Throughout this I see a variable named "relativePaths" and I imagine many people would think "relative to what though?"

Maybe it could be called `rootRelativePaths` or something else slightly more decriptive.

Reply:
Thanks for pointing it out. I ended up reverting introduction of the class altogether. See the previous comment